notes_for_class/source/index.rst

```diff
@@ -1,27 +0,0 @@
-
```

Comment:
you shouldn't mess with anything here -- particularly not deleting it!

see if a git pull upstream master brings it back. Otherwise, try:

git checkout 4464705893b26f notes_for_class/source/index.rst

(after the git pull upstream master)